### ISSUE TITLE
Complete system crash when using the broken files search

### ISSUE THREAD

**Issue by 1337-Haxx0r** [qarmin/czkawka#917]
Hardware:

CPU: 4 Cores / 8 Threads
RAM: 8 GB

Affected Versions / Tested on:

Linux Mint 21.1 with GTK3, Czkawka-Version 4.1.0 (linux_czkawka_gui)
Debian Bookworm SID (12) with GTK4, Czkawka-Version 5.0.2 (linux_czkawka_gui)
Debian Bookworm SID (12) with GTK4, Czkawka-Version 5.1.0 (linux_czkawka_gui)

When using the Broken Files Tab the Host Machine completely crashes sometimes. It completely shuts down.
All other options seem to work fine, even when using more than a million files.
But as soon as I use about over 20000 files in the Broken Files Tab, it crashes.
Sometimes it freezes at about 99% or 100% (in the lower status bar) and after a few seconds it also crashes.
First I thought it could be an Integer Overflow or something, but this can't be the case since the bug also occurs when using 279 files.
I also tried running czkawka 5.0.2 inside a Virtual Machine. It also crashes, but not just the Guest-VM, it crashed the whole Host-System!
It completely shut it down. Hopefully not a security vulnerability.

System resources seem fine right before the crash: 20% RAM usage, All 8 Threads on 100% as expected.
Temperature of the Cores also seem fine: 85 °C. Shutdown temperature is 100 °C.

**Comment by qarmin** [comment]
If app runs with user privileges and crashes entire OS, then I think that problem is somewhere else.

I suggest to look at journals, run cpu stress test or cpu hardware diagnose tool(I only found intel app, that works with wine - https://www.intel.com/content/www/us/en/download/15951/19792/intel-processor-diagnostic-tool.html) to see that problem is not with hardware

**Comment by 1337-Haxx0r** [comment]
Thanks, you were right, it was indeed an issue with the cpu cooling. It did reach the 100° C on all cores. I had to open the device and reapply some thermal paste onto the cpu. Also there was a lot of dust in the fan, therefore I cleaned it and closed the device again. Now everything runs fine. This explains why my fan was always on 3000 RPM, even when the OS was idling. Now the fan is less annoying and the cpu temperature stays below 60° C most of the time.
